lib/monitor.js

```diff
@@ -208,3 +206,17 @@ module.exports = internals.Monitor = class {
         }
     }
 };
+
+const setupPipeline = (streams, onFinished) => {
+
+    const stream = streams[0];
+
+    if (streams.length === 1) {
+        finished(stream, onFinished);
```

Comment:
Nice, didn't know about the `finished()` helper!

Reply:
OMG, finally it's provided out-of-the-box